src/commands/runtime/activation/logs.js

```diff
@@ -55,18 +83,44 @@ ActivationLogs.args = [
 
 ActivationLogs.flags = {
   ...RuntimeBaseCommand.flags,
+  action: flags.string({
+    description: 'Fetch logs for a specific action',
+    default: '',
+    exclusive: ['manifest, package'],
+    char: 'a'
+  }),
+  manifest: flags.boolean({
+    description: 'Fetch logs for all actions in the manifest',
+    exclusive: ['package, action'],
+    char: 'm'
+  }),
+  package: flags.string({
+    description: 'Fetch logs for a specific package in the manifest',
+    exclusive: ['manifest, action'],
+    char: 'p'
+  }),
+  deployed: flags.boolean({
+    description: 'Fetch logs for all actions deployed under a specific package',
+    exclusive: ['manifest, action'],
+    char: 'd'
+  }),
   last: flags.boolean({
     char: 'l',
     description: 'retrieves the most recent activation logs'
   }),
   strip: flags.boolean({
     char: 'r',
-    description: 'strip timestamp information and output first line only'
+    description: 'strip timestamp information and output first line only',
+    default: false
+  }),
+  limit: flags.integer({
+    description: 'return the last `limit` activation logs. Max 50',
+    exclusive: ['last']
   }),
-  count: flags.integer({
-    char: 'c',
-    description: 'used with --last, return the last `count` activation logs. Max 5',
-    default: 1
+  tail: flags.boolean({
```

Comment:
I think `--watch & -w` are more approachable, should we have aliases?

Reply:
Good point. I wonder if `poll` would be a good verb here. Note that wsk uses `poll` too